processor/attributesprocessor/README.md

```diff
@@ -105,11 +106,11 @@ processors:
 Refer to [config.yaml](./testdata/config.yaml) for detailed
 examples on using the processor.
 
-## Include/Exclude Spans
+## Include/Exclude Filtering
 
 The [attribute processor](README.md) and the [span processor](../spanprocessor/README.md) expose
```

Comment:
I'd say we should remove span processor link from here to avoid confusion

Reply:
Agreed. I also updated the example config with the rest of the supported fields.